src/styles/reset.js

```diff
@@ -0,0 +1,197 @@
+import { css } from 'styled-components';
+
+export default css`
+  /* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Fyi an alternative is normalize.css. you can install it via npm and simply import it somewhere in your js code